client/src/components/Verejne/EntitySearch/index.js

```diff
@@ -33,6 +33,7 @@ const EntitySearch = ({
   entitySearchEids,
   entitySearchFor,
 }) => {
+  const entitySearchEidsConst = entitySearchEids === undefined ? 0 : entitySearchEids.length
```

Comment:
just a note - this shouldn't be needed at all and should be handled by the component being wrapped in a data-provider - but that's a problem unrelated to this PR, so it's an ok addition for the time being